istioctl/cmd/proxystatus_test.go

```diff
@@ -24,11 +24,11 @@ func TestProxyStatus(t *testing.T) {
 	cases := []execTestCase{
 		{ // case 0
 			args:           strings.Split("proxy-status", " "),
-			expectedString: "NAME     CDS     LDS     EDS     RDS     ISTIOD",
+			expectedString: "NAME     CDS     LDS     EDS     RDS     NDS     ECDS     ISTIOD",
```

Comment:
a bit worried this to too wide for most consoles. The current `ps` just barely fits on my 1440p monitor in fullscreen today

Reply:
Ah! I see. It looks ugly when split over multiple lines. **For the moment I added `-A` flag (or `--all-xds-statuses`) to show additional columns**. So `istioctl proxy-status` still shows old output. If required it is straightforward to add something like `--xds-status-cols=nds,ecds,pcds` to show only those three selected columns. But I think it is unnecessary at the moment.